cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -428,9 +436,11 @@ def update_kubelet_status():
 
 
 @when('certificates.available', 'kube-control.connected')
-def send_data(tls, kube_control):
+def send_data():
     '''Send the data that is required to create a server certificate for
     this server.'''
+    kube_control = endpoint_from_flag('kube-control.connected')
```

Comment:
Any chance when you call this manually that kube-control.connected isn't set?

Reply:
Yes, that is entirely possible.  Thanks.